src/rollingBall.ts

```diff
@@ -23,7 +23,15 @@ import { xMean, xMaxValue, xMinValue } from 'ml-spectra-processing';
  * @param {Number} [options.windowM] - width of local window for minimization/maximization, defaults to 4% of the spectrum length
  * @param {Number} [options.windowS] - width of local window for smoothing, defaults to 8% of the spectrum length
  */
-export function rollingBall(spectrum, options = {}) {
+
+interface Options {
+  windowM: number;
+  windowS: number;
+}
+export function rollingBall(
+  spectrum: Float64Array,
```

Comment:
Please use the `NumberArray` utility type from the `cheminfo-types` package. When possible, always let TS infer the types for you.